src/main/java/jp/skypencil/errorprone/slf4j/ManuallyProvidedMessage.java

```diff
@@ -35,17 +33,10 @@ public class ManuallyProvidedMessage extends BugChecker implements MethodInvocat
 
   @Override
   public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState state) {
-    Symbol method = ASTHelpers.getSymbol(tree.getMethodSelect());
-    String methodStr = method.toString();
-    String methodName = methodStr.substring(0, methodStr.indexOf('('));
-    if (!Consts.TARGET_METHOD_NAMES.contains(methodName)) {
+    if (!Consts.IS_LOGGING_METHOD.matches(tree, state)) {
       return Description.NO_MATCH;
     }
 
-    Symbol clazz = method.enclClass();
-    if (!"org.slf4j.Logger".equals(clazz.toString())) {
```

Comment:
`IS_LOGGING_METHOD` also matches for subtypes of Logger now...
but this would now be consistent with behavior in `FormatShouldBeConst` and `SignOnlyFormat`